### ISSUE TITLE
Undefine message when submitting the Newsletter form

### ISSUE THREAD

**Issue by purplecodes** [Codeinwp/elementor-addon-widgets#51]
User is getting an undefine message after entering email on the Newsletter form.
https://www.transamericanheroes.com/sizzify-failure/
screenshot: https://prnt.sc/m2qt8c

Ticket link: https://wordpress.org/support/topic/undefined-error-on-newsletter-form/#post-11049967

Steps taken:
- installed poopy.life
- installed Elementor and Sizzify Lite only
- created a page and edited it with Elementor
- added a section for the Newsletter Form widget
- entered API key and List ID from Mailchimp
- tested and got the error.
- changed the type from email to text, same issue -->> https://prnt.sc/m2qu0j

**Comment by purplecodes** [comment]
@contactashish13 kindly look into this when you can? Let me know if you need more information.

**Comment by mileamadalin** [comment]
+1 Can you fix this one? Our customers complain about this one 

@contactashish13

**Comment by mileamadalin** [comment]
@radu-themeisle can we please make the release here?

**Comment by radu-themeisle** [comment]
Yes, but first we need to fix some more issues which I missed on the first test. Hopefully at the beginning of next week we'll have a release.
ref codeinwp/themeisle-content-forms#23
